### ISSUE TITLE
moloch-capture -r does not honor umask of calling user's shell

### ISSUE THREAD

**Issue by philhagen** [aol/moloch#976]
**Elasticsearch version**: 5.6.14

**Moloch version**: 1.6.2

**OS name and version**: Lentos 7.6.1810

**How was Moloch built/installed:** rpm

Issue: Running `moloch-capture` seems to always create files in `/data/moloch/raw/` with umask of `0007`, rather than that of the calling user account's shell settings.  In the example below, I would expect the files should be set with permissions of `-rw-rw-r--`, per the umask setting of `0002`.

```
[moloch_user@for572-moloch ~]$ whoami
moloch_user
[moloch_user@for572-moloch ~]$ umask
0002
[moloch_user@for572-moloch ~]$ moloch-capture -q --copy -r eth1_internet.pcap -t lab4.3 -t eth1
[moloch_user@for572-moloch ~]$ touch /data/moloch/raw/fff
[moloch_user@for572-moloch ~]$ ls -l /data/moloch/raw/
total 1248640
-rw-rw-r-- 1 moloch_user moloch_user         0 Dec 17 12:15 fff
-rw-rw---- 1 moloch_user moloch_user 642449016 Dec 17 12:14 for572-moloch-171106-00000001.pcap
-rw-rw---- 1 moloch_user moloch_user 636133986 Dec 17 12:14 for572-moloch-171106-00000002.pcap

**Comment by awick** [comment]
Correct, we don't want to ever allow "other".  What issue are you hitting?

**Comment by philhagen** [comment]
right now, when I load via a user account (such as `moloch_user` in this case), the process running viewer, aka `moloch` can't see the pcap, so the UI shows "no pcap available.

I can fix this by chmodding `/data/moloch/raw/` to `3777`, so users can remove files they created themselves (just like `/tmp/`, which is chmodded to `1777`, and so the group that owns `/data/moloch/raw/` itself is allocated to all files created inside that directory.  Then, if the directory itself is chows'ed to `root:moloch`, any non-root user can run `moloch-capture` per above, can then remove their own files at the command line if needed, and the `moloch` user running viewer can see the content.

If this is considered the preferred path - no worry.  It was just unexpected behavior, hence the report.  Thanks for the perspective!

**Comment by awick** [comment]
Is this in raw capture or --copy mode?  Maybe --copy could not force that?  Would that fix your issue?

**Comment by philhagen** [comment]
this is with `--copy`.  If this behavior was not exhibited with that flag, then yes, that would be ideal.
